src/interactivity/win32/ut_interactivity_win32/UiaTextRangeTests.cpp

```diff
@@ -695,6 +695,30 @@ class UiaTextRangeTests
                     {0, bottomRow}
                 }
             }
+
+            MoveTest{
+                L"can move to a new row when necessary when moving forward",
+                { lastColumnIndex, 0 },
+                { lastColumnIndex, 0 },
+                5,
+                {
+                    5,
+                    {lastColumnIndex , 0 + 5},
+                    {lastColumnIndex , 0 + 5}
+                }
+            },
+
+            MoveTest{
+                L"can move to a new row when necessary when moving backward",
+                { 0, 0 + 1 },
+                { lastColumnIndex, 0 + 1 },

```

Comment:
2 things:
1. you're going to want these two coordinates to be the same value. That way we have a degenerate range.
2. Since we're moving backwards 5 times, you'll want this to have a y-value greater than 5. That way we don't hit the buffer boundary and stop moving.